about.html

```diff
@@ -63,8 +63,10 @@ <h3>Creative Commons</br>Global Summit 2015 Logo</h3>
             <h3>See the past CC Global Summits</h3>
             <div class="separator"></div>
             <p>Take a look at the last two CC GLobal Summits which took place in Buenos Aires and in Warsaw.</p>
-            <div class="past1"><img src="./images/past1.png" alt="NFT collections"></div>
-            <div class="past2"><img src="./images/past2.jpg" alt="NFT collections"></div>
+            <div class="containerPast">
+                <div class="past1"><img src="./images/past1.png" alt="NFT collections"></div>
+                <div class="past2"><img src="./images/past2.jpg" alt="NFT collections"></div>
+            </div>
```

Comment:
- Great job adding images about past global submits, but I think you may have forgotten to include the `YEAR` and `WHERE` the submit was held ON the images. The images on Behance also have a background color of `#ee523c`. Can we add that too?

Here is what I can see compared to Behance 👇🏽 
| Your image for past submit  |
| :---:   | 
|![nftbad](https://user-images.githubusercontent.com/99246359/203655416-93c22ee6-7fd6-4f34-b06b-59c466dbbb1e.PNG)  |

|Behance past submit |
| :---:   | 
|![nftgood](https://user-images.githubusercontent.com/99246359/203655456-797d11bb-a411-49e6-b0ed-cc180d9cd999.PNG)  |

What do you think about this suggestion?? It is important for our design to follow cindy shin's behance template as much as possible 🌸